feature-utils/poly-look/src/react-components/filterChips.jsx

```diff
@@ -1,5 +1,5 @@
 import React, { useState } from "react";
-import Chip from "../chip/chip.jsx";
+import Chip from "./chip.jsx";
```

Comment:
Main problem I see here is that coverage is not super-high, at 43.47%. Still, that can be done later on, and it's a great start.

Reply:
(this as a reaction to your "confused" emoji) Generally, coverage should be above 80%, so that most code paths are checked, especially corner cases. But please just pick up the fact that it's a great start, which it is. We can add more tests as necessity arises; since you've added coverage testing, identifying those parts is not difficult and can be done later.